requirements.txt

```diff
@@ -8,7 +8,7 @@ xarray
 numpy
 zarr
 cftime
-git+https://github.com/mpi4py/mpi4py.git@aac3d8f2a56f3d74a75ad32ac0554d63e7ef90ab
```

Comment:
We might need to stick with `mpi4py` master based on experiments, but since we still do not have halo exchanges working on daint, this is probably fine for now.

Reply:
This was changed because master was breaking the docker build using the cuda image base ... but a lot has changed since then. I can try again to see if this works.

Reply:
It works now yay!